src/autocomplete/index.jsx

```diff
@@ -23,13 +26,24 @@ export default function AutoComplete(props) {
     };
   });
 
-  function suggest (query, syncResults) {
+  function simpleSearch (query, syncResults) {
     syncResults(query
       ? props.options.filter(result => result.label.toLowerCase().includes(query.toLowerCase()))
       : []
     );
   }
 
+  function apiSearch (query, syncResults) {
+    if (!query) {
+      return syncResults([]);
+    }
+    fetch(`${apiPath}/get-autocomplete-options/${query}`).json
```

Comment:
Having `/get-autocomplete-options/` hard-coded into the url feels a little wrong.